docs/application/web/get-started/addon/first-addon.md

```diff
@@ -0,0 +1,521 @@
+    const options = {
+        type: 'info',
+        buttons: ['OK'],
+        title: 'Resume',
+        message: 'Hello. Resume occurred!!'
+    };
+    addonAPI.window.showMessageBox(winId, options);
+}
+```
+    
+
+After using addonapi and adding the code, the JavaScript file looks like:
+
+![Code for resume pop-up](media/hello_resume_code.png)
+
+Following are the steps to confirm if the changed file works properly:
+
+1. With the changed file, you need to build a signed package again.
+
+2. Before installing the newly built package, you have to remove the installed addon first.
+
+3. In Addon Settings App, you can see the **X** button in red box in the row of the installed addon list. To remove the installed addon, you need to press the button.
+
+4. Now, the changed addon package with pop-up on resume event will be installed if you follow the steps you did with the template package before.
+
+5. You run Basic UI Web application. You will see the pop-up as below:
+
+**Figure: Addon Pop-up on Resume Event in Basic UI**
+
+![Showing pop-up in Basic UI](media/hello_resume.png)
+
+
+### Loading HTML file from addon
+
+In an addon, you can load your own HTML file. In this section, an example of an HTML file that shows `Hello, I'm an Addon` is explained.
+
+Ensure that all the files related to an addon are located under `shared\res` of your addon project. Therefore, the added HTML file must also be located in the same directory.
+
+![Add a new file](media/new_file.png)
+
+![Generate the HTML file](media/generate_hello_html.png)
+
+Following is an example of `hello.html` HTML file:
+
+```
+<!DOCTYPE html>
+<html>
+<head>
+    <meta name="viewport" content="user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0, width=device-width">
+</head>
+<body>
+    <h1>Hello, I'm an Addon</h1>
+</body>
+</html>
+```
+
+The HTML file should be loaded in the addon. To do that, you can use `addonapi.window.loadURL()` when the resume event occurs. Then, the `rusume()` will look like:
+
+```
+resume(winId) {
+    console.log('TmplDefault.resume');
+    addonAPI.window.loadURL(winId, 'file://' + __dirname + '/hello.html');
+    const options = {
+        type: 'info',
+        buttons: ['OK'],
+        title: 'TcResume',
+        message: 'Hello. Resume occurred!!'
+    };
+    addonAPI.window.showMessageBox(winId, options);
+}
+```
+
+You need to remove the installed addon, build a package with the HTML file and changed JavaScript file, and install the new package.
+
+If there's no issue, you will see the screen as below:
+
+![Resume with HTML and pop-up](media/resume_html_popup.png)
+
+### Return to main page of Web application
+
+In the implementation above, it is impossible to return to the main page of the Web application because it loads `hello.html` whenever the resume event happens.
+
+You have 2 issues to resolve the problem:
+
+- Showing the HTML page in every resume event
+- No way to return to the main page of the Web application
+
```

Comment:
I did not mean to select one of two options. Therefore, I have changed as below:

When you implement to [load HTML file from addon](#load-html-from-addon), you will not be able to return to the main page of the Web application. This is because it loads `hello.html` in every resume event and does not know the URL of the main page of the Web application. To return to the main page, you have to resolve these two issues.


Please verify